### DISCUSSION TITLE
Writing in several languages in Android 10

### DISCUSSION THREAD

**Discussion by ariasuni** [AnySoftKeyboard/AnySoftKeyboard#2912] [Q&A]
I want to be able to type in both English and French with the correct spell checker. In the past, I setup French and English in Settings → System → Languages & input → Languages, and French bépo as my layout, and then I could change between «Français AnySoftKeyboard» and «English AnySoftKeyboard» and it would change the spell check accordingly.

I stopped working in Android 10, even though I didn’t change my settings: the spell check stays in French (the first language in Android settings). Is this an Android thing, something from AnySoftKeyboard, or is my rom broken somehow? GBoard or FlorisBoard seem to be able to use both dictionaries at the same time which would be nice.

Edit: my problem may be [this LineageOS bug](https://gitlab.com/LineageOS/issues/android/-/issues/1723), but still, why is it working (differently but both are fine for me) with other keyboards?

**Comment by cauerego** [comment]
thanks for mentioning florisboard! 😀

(sorry this is no answer)

**Reply by ariasuni** [reply]
I discovered your application because it has bépo! Just created a section on our wiki btw: https://bepo.fr/wiki/Android#FlorisBoard.

**Reply by cauerego** [reply]
it's not my application 🤣

i just enjoyed finding about it.
